lib/themes-panel.coffee

```diff
@@ -147,6 +153,29 @@ class ThemesPanel extends View
     @activeUiTheme = @getActiveUiTheme()
     @activeSyntaxTheme = @getActiveSyntaxTheme()
     @populateThemeMenus()
+    @toggleCurrentThemeButtons()
+    @handleCurrentThemeButtons()
+
+  handleCurrentThemeButtons: ->
+    @currentUiThemeSettings.on 'click', (event) =>
+      event.stopPropagation()
+      @parents('.settings-view').view()?.showPanel(@activeUiTheme, {back: 'Themes', pack: @activeUiTheme})
+
+    @currentSyntaxThemeSettings.on 'click', (event) =>
+      event.stopPropagation()
+      @parents('.settings-view').view()?.showPanel(@activeSyntaxTheme, {back: 'Themes', pack: @activeSyntaxTheme})
+
+  toggleCurrentThemeButtons: ->
+    if not atom.config.get(@activeUiTheme)?
```

Comment:
Typically we use `unless` instead of `if not` since it is functionally the same.

But in this case I think it would make sense to just make this a regular `if` / `else` and swap the order:

```coffee
if atom.config.get(@activeUiTheme)?
  @currentUiThemeSettings.show()
else
  @currentUiThemeSettings.hide()
```